packages/manager/modules/netapp/src/dashboard/volumes/dashboard/acl/template.html

```diff
@@ -0,0 +1,129 @@
+<div>
+    <h2>
+        <span data-translate="netapp_volumes_acl"></span>
+        (<abbr
+            data-translate-attr-title="netapp_volumes_acl_title"
+            data-translate="netapp_volumes_acl_title_abbr"
+        ></abbr
+        >)
+    </h2>
+    <p data-translate="netapp_volumes_acl_description"></p>
+    <oui-button
+        variant="secondary"
+        disabled="$ctrl.isAdding"
+        on-click="$ctrl.addAcl()"
+        icon-left="oui-icon-add"
+        class="mb-3"
+    >
+        <span data-translate="netapp_volumes_acl_add"></span>
+    </oui-button>
+    <oui-datagrid rows="$ctrl.acls" ng-if="$ctrl.acls.length">
```

Comment:
```suggestion
    <oui-datagrid rows="$ctrl.acls" >
```
You could still display the datagrid but empty no ?

Reply:
Assuming that there's no `aclRule` already configured, the expected behavior is
that we have to hide the datagrid. Once the user click on "Add a new ACL", then
we have to display the datagrid.